board/aspeed/evb_ast2700/evb_ast2700.c

```diff
@@ -325,6 +330,36 @@ int set_board_info(const u8* scm_eeprom_buf, const u8* hpm_eeprom_buf)
 	return 0;
 }
 
+void mount_safs_spi_mux()
+{
+	printf("configuring gpios for eDAF ...\n");
+	/* mount P0 BIOS SPI */
+	run_command("gpio set 169", 0); //GPIO V1
```

Comment:
int run_command(const char *cmd, int flag); - please handle return , will help for debug incase of failure.